src/main/java/teammates/ui/controller/InstructorFeedbacksPageData.java

```diff
@@ -8,40 +9,273 @@
+            String href = getFeedbackSessionStatsLink(session.courseId, session.feedbackSessionName);
+            
+            String recent = "";
+            if (session.isOpened() || session.isWaitingToOpen()) {
+                recent = " recent";
+            } else if (displayedStatsCount < InstructorFeedbacksPageData.MAX_CLOSED_SESSION_STATS
+                       && !TimeHelper.isOlderThanAYear(session.createdTime)) {
+                recent = " recent";
+                ++displayedStatsCount;
+            }
+            
+            String actions = "";
+            try {
+                actions = getInstructorFeedbackSessionActions(session, false, instructors.get(courseId),
+                                                getCourseIdSectionNamesMap(sessions).get(courseId));
+            } catch (EntityDoesNotExistException e) {
+                // nothing
+            }
+            
+            ElementTag elementAttributes ;
+            if (session.courseId.equals(courseIdForNewSession) && session.feedbackSessionName.equals(feedbackSessionNameForSessionList)) {
+                elementAttributes = new ElementTag("class", "sessionsRow warning");
+            } else {
+                elementAttributes = new ElementTag("class", "sessionsRow");
+            }
+            
+            rows.add(new FeedbackSessionRow(courseId, name, tooltip, status, href, recent, actions, elementAttributes));
+        }
+        
+        return rows;
+    }
+    
+    public FeedbackSessionsList getFsList() {
+        return fsList;
+    }
+    
+    public FeedbackSessionsNewForm getNewForm() {
+        return newForm;
+    }
+
+    public ArrayList<ElementTag> getTimeZoneOptionsAsHtml(FeedbackSessionAttributes fs){
+        return getTimeZoneOptionsAsElementTags(fs == null ? 
+                                               Const.DOUBLE_UNINITIALIZED : 
+                                               fs.timeZone);
+    }
+
 
+    public ArrayList<String> getGracePeriodOptionsAsHtml(FeedbackSessionAttributes fs){
+        return getGracePeriodOptionsAsHtml(fs == null ? 
+                                           Const.INT_UNINITIALIZED : 
+                                           fs.gracePeriod);
+    }
+    
+    public ArrayList<ElementTag> getGracePeriodOptionsAsElementTags(FeedbackSessionAttributes fs) {
+        return getGracePeriodOptionsAsElementTags(fs == null ? 
+                                                  Const.INT_UNINITIALIZED : 
+                                                  fs.gracePeriod);
+    }
+    
+    public ArrayList<ElementTag> getFeedbackSessionTypeOptions(String defaultSessionType) {
+        ArrayList<ElementTag> result = new ArrayList<ElementTag>();
+        
+        ElementTag standardFeedbackSession = createOption("Session with your own questions", "STANDARD", defaultSessionType == null ||
+                                                          defaultSessionType.equals("TEAMEVALUATION"));
```

Comment:
The boolean condition is not obviously connected, you may want to put them both on a single line.
Also, operators should be at the start of new lines, not at the end of the previous.